addons/html_builder/static/src/builder/options/font_awesome_option.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+<t t-name="html_builder.FontAwesomeOption">
+    <BuilderRow label.translate="Color">
+        <BuilderColorPicker styleAction="'color'"/>
+    </BuilderRow>
+
+    <BuilderRow label.translate="Background Color">
+        <BuilderColorPicker styleAction="'backgroundColor'"/>
+    </BuilderRow>
+
+    <BuilderRow label.translate="Size">
+        <BuilderButtonGroup>
+            <BuilderButton classAction="''" title="'Size 1x'">1x</BuilderButton>
+            <BuilderButton classAction="'fa-2x'" title="'Size 2x'">2x</BuilderButton>
+            <BuilderButton classAction="'fa-3x'" title="'Size 3x'">3x</BuilderButton>
+            <BuilderButton classAction="'fa-4x'" title="'Size 4x'">4x</BuilderButton>
+            <BuilderButton classAction="'fa-5x'" title="'Size 5x'">5x</BuilderButton>
+
+            <!-- TODO: nothing use it ??? Hidden buttons which allows to automatically remove other fa
+                sizing class we do not suggest.
+                TODO: implement a param to add those extra classes to remove
+                via selectClass (also fixes the fact that nothing is selected
+                if fa-1x is actually used on the icon in this case) -->
+            <!-- <BuilderButton classAction="'fa-1x'" dependencies="'fake'"/>
+            <BuilderButton classAction="'fa-lg'" dependencies="'fake'"/> -->
```

Comment:
but you still need to remove those classes at the 'clean' or I missed something?